index.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./assets/css/styles.css" />
+
+    <title>Jonathan Jefferies Portfolio</title>
+  </head>
+
+<body>
+
+<nav id="navbar" class="nav">
+  <ul class="nav-list">
+    <li>
+      <a href="#aboutme-section">About Me</a>
+    </li>
+    <li>
+      <a href="#projects">Projects</a>
+    </li>
+    <li>
+      <a href="#contact">Contact</a>
+    </li>
+  </ul>
+</nav>
+
+<section id="aboutme-section" class="aboutme-section">
+  <h1>Jonathan Jefferies Portfolio</h1>
+<div class="profile-image">
+  <img src="./assets/images/jonny.jpeg" alt="Jonny and Hannah test">
```

Comment:
This image does not appear on the portfolio as the file path is not correct

Reply:
changeed path